pydantic/utils.py

```diff
@@ -277,16 +277,25 @@ def to_camel(string: str) -> str:
 T = TypeVar('T')
 
 
-def unique_list(input_list: Union[List[T], Tuple[T, ...]]) -> List[T]:
+def unique_list(
+    input_list: Union[List[T], Tuple[T, ...]],
+    *,
+    get_name: Callable[..., str] = lambda x: str(x),
```

Comment:
```suggestion
    name_factory: Callable[[T], str] = str,
```
?

Reply:
No need to be sorry for the change. Way better name, default and typing 😄